application/config/config.php

```diff
@@ -1,22 +1,11 @@
-<?php if (!defined('BASEPATH')) exit('No direct script access allowed');
+<?php
+defined('BASEPATH') OR exit('No direct script access allowed');
```

Comment:
Strange that this file doesn't allow direct script access but also sets the executable bit.

Reply:
In the CI distribution, all files have the x bit set. It might just be because CI apps also get run under Windows, so rather than deal with perm issues they just defend against direct access in code.